client/blocks/login/two-factor-authentication/verification-code-form.jsx

```diff
@@ -111,31 +136,38 @@ class VerificationCodeForm extends Component {
 							) }
 						</FormFieldset>
 
-						<FormButtonsBar>
-							<FormButton
-								onClick={ this.onSubmit }
-								primary
-								disabled={ this.props.isRequestingTwoFactorAuth }
-							>{ translate( 'Log in' ) }</FormButton>
-						</FormButtonsBar>
+						<FormButton
+							className="two-factor-authentication__form-button"
+							onClick={ this.onSubmit }
+							primary
+							disabled={ this.props.isRequestingTwoFactorAuth }
+						>{ translate( 'Continue' ) }</FormButton>
 					</Card>
-				</form>
 
-				<p>
-					<ExternalLink
-						icon={ true }
-						target="_blank"
-						href="http://en.support.wordpress.com/security/two-step-authentication/">
-						{ translate( 'Help' ) }
-					</ExternalLink>
-				</p>
-
-				<hr />
-				{ isSmsSupported && (
-					<p>
-						<a href="#" onClick={ this.sendSmsCode }>{ translate( 'Send recovery code via text' ) }</a>
-					</p>
-				) }
+					<Card className="two-factor-authentication__form-action is-compact">
+						<p>
+							{ translate( 'Or continue to your account using:' ) }
+						</p>
+
+						{ twoFactorAuthType === 'code' && isSmsSupported && (
+							<p>
+								<a href="#" onClick={ this.sendSmsCode }>{ translate( 'Code Via Text Message' ) }</a>
+							</p>
+						) }
+
+						{ twoFactorAuthType === 'sms' && isAuthenicatorSupported && (
+							<p>
+								<a href="#" onClick={ this.verifyWithAuthenticator }>{ translate( 'An Authenticator Application' ) }</a>
+							</p>
+						) }
+
+						{ isPushSupported && (
+							<p>
+								<a href="#" onClick={ this.verifyWithPush }>{ translate( 'The WordPress Mobile App' ) }</a>
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'WordPress Mobile App' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).